spec/controllers/works_controller_spec.rb

```diff
@@ -97,13 +97,13 @@
             @work2.index.refresh
           end
 
-          xit "should not show restricted works to guests" do
+          it "should not show restricted works to guests" do
             get :index, tag_id: @fandom.name
             expect(assigns(:works).items).to include(@work)
             expect(assigns(:works).items).not_to include(@work2)
           end
 
-          xit "should show restricted works to logged-in users" do
+          it "should show restricted works to logged-in users" do
```

Comment:
This is the test that breaks the import specs (I enabled it locally to check and the `story_parser` tests failed). I suspect it's due to the `fake_login` method which amongst other things, stubs the `current_user` call to return a user who isn't an archivist, and then doesn't reset normal calls afterwards. I think this test needs to clean up after itself.